Source/Particles/Collision/MCCProcess.cpp

```diff
@@ -65,7 +65,6 @@ MCCProcess::init (const std::string& scattering_process, const amrex::Real energ
 #ifdef AMREX_USE_GPU
     m_exe_d = m_exe_h;
     m_sigmas_d.resize(m_sigmas_h.size());
```

Comment:
Accidentally removed, unrelated code:
```suggestion
    m_sigmas_d.resize(m_sigmas_h.size());
    m_exe_d.m_sigmas_data = m_sigmas_d.data();
```